src/pymor/discretizers/builtin/gui/matplotlib.py

```diff
@@ -87,8 +100,13 @@ def __init__(self, figure, grid, bounding_box=None, U=None, vmin=None, vmax=None
         self.colorbar = colorbar
         self.animate = self.set
 
+        if bounding_box is None:
+            bounding_box = grid.bounding_box()
+        assert len(bounding_box) == 2 and all(len(b) == 2 for b in bounding_box)
```

Comment:
Currently fails in `make docker_docs`: https://zivgitlab.uni-muenster.de/pymor/pymor/-/jobs/463616